runtime/src/snapshot_utils.rs

```diff
@@ -395,6 +396,25 @@ pub fn archive_snapshot_package(
         ("duration_ms", timer.as_ms(), i64),
         ("size", metadata.len(), i64)
     );
+
+    info!("Create snapshot local file in {:?}", tar_dir);
+    create_snapshot_local_file(tar_dir)?;
+    Ok(())
+}
+
+pub fn has_snapshot_local_file(snapshot_archive_dir: &Path) -> bool {
```

Comment:
nit: Please use `snapshot_archives_dir` (archive**S**, to be explicit)
(and in the other locations)